src/components/RegisterLoginModal/RegisterLoginModal.jsx

```diff
@@ -1,15 +1,34 @@
-import { useState } from 'react';
-
+/* eslint-disable react/no-children-prop */
+/* eslint-disable no-shadow */
```

Comment:
I think the 2 rules above should be removed and the code causing the eslint errors should be changed. I'll let Tomek know what needs to be changed.